src/views/index.tsx

```diff
@@ -1,19 +1,13 @@
-import * as React from "react";
```

Comment:
We should probably keep the Root view as simple as possible, abstracting as much component logic to child components so we have an end result like.
```
<div className="root">
    <Navbar />
    <Canvas />
</div>
```

Reply:
Agreed. I was using that wrapper around the Grid for debugging. Added it as a temp style to the `Grid.scss`